apps/avatax/src/modules/avatax/avatax-errors-parser.ts

```diff
@@ -0,0 +1,41 @@
+import * as Sentry from "@sentry/nextjs";
+import { z } from "zod";
+import { InvalidAppAddressError } from "../taxes/tax-error";
+import { normalizeAvaTaxError } from "./avatax-error-normalizer";
+
+export class AvataxErrorsParser {
+  private static schema = z.object({
+    // https://developer.avalara.com/avatax/errors/
+    code: z.enum(["InvalidAddress"]),
+    details: z.array(
+      z.object({
+        faultSubCode: z.string().optional(),
```

Comment:
**Question:** do we need to catch `faultSubCode`? I'm thinking if catching all `InvalidAddress` errors are not enough? We can see what went wrong in logs. Also I don't see `faultSubCode` in Avatax docs: https://developer.avalara.com/avatax/errors/InvalidAddress/